src/Template/Element/index/bulk_actions/form_end.ctp

```diff
@@ -0,0 +1,30 @@
+<?php
+if (empty($bulkActions)) {
+    return;
+}
+
+$this->Form->templates([
+    'submitContainer' => '{{content}}',
+]);
+
+$submitButton = $this->Form->submit(__d('crud', 'Apply'), [
+    'class' => 'btn btn-success btn-bulk-apply',
+    'div' => false,
```

Comment:
There's no `div` option. It would be better to consistently use `FormHelper::button()` everywhere.